cmd/ctr/shim.go

```diff
@@ -133,11 +137,15 @@ var shimStartCommand = cli.Command{
 	Name:  "start",
 	Usage: "start a container with a shim",
 	Action: func(context *cli.Context) error {
-		service, err := getShimService()
+		var (
```

Comment:
I think the var blocks are fine here, there is more than one declaration inside and we need the same context for all requests going forward in the future